app/components/dev/DevDebug.tsx

```
@@ -12,30 +12,30 @@ import { BaseButton } from "@/app/components";
 // Variables
 import { FORM_FILL_VALUES } from "@/lib/variables";
 
-type DevDebugProps = {};
+type DevDebugProps = Record<string, never>;
 
 const DevDebug = ({}: DevDebugProps) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)